WORKSPACE.bazel

```diff
@@ -17,9 +17,9 @@ workspace(name = "stablehlo")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-LLVM_COMMIT = "00128a20eec27246719d73ba427bf821883b00b4"
+LLVM_COMMIT = "3f9cabae0029bcbe88835aaa4c417ce41e584fb1"
 
-LLVM_SHA256 = "9fff2ccb6c262f3d5e2f98c281a0b99a585daee83742e1599709ff61cfc222af"
+LLVM_SHA256 = "626e1c8e491cd70ef540c403fdd43b9ff9ee50aafe181c32b14e67f715ad015e"
```

Comment:
Revert? I'd guess we're ahead of this pin by now.

Reply:
Done.